test/lib/get-downloader-spec.js

```diff
@@ -88,8 +92,10 @@ describe('#getDownloader', () => {
     await downloader(null, command);
     await unlockConfigFiles('.application.yml');
     await deferred.promise;
-    message.should.equal(`${pad(c.green('downloaded'), 13)}\tviews/layouts/Example Layout.hbs`);
+    message.should.equal(`${pad(c.green('downloaded'), 13)}\texperience/layouts/GET mynewroute.hbs`);
     spy.restore();
+    // (await pathExists('./experience/layouts/Get mynewroute.hbs')).should.be.true();
+    // (await readFile('./experience/layouts/Get mynewroute.hbs')).toString().should.equal('a body');
```

Comment:
some random commented out code?

Reply:
well I was actually checking that the filenames were created with the correct name but you cant do this on the CI.